index.html

```diff
@@ -4,7 +4,7 @@
 <head>
 	<meta charset="UTF-8" />
 	<title>Fourmation - Enabling the connected world</title>
-	<meta content="width=device-width,initial-scale=1.0,maximum-scale=1.0" name="viewport">
+	<meta content="width=device-width,initial-scale=1.0,maximum-scale=5.0" name="viewport">
```

Comment:
Does not seem to make any difference on the site. It's possible to zoom on both desktop and mobile, even with maximum-scale=1.0